app/locales/en/translations.js

```diff
@@ -191,11 +191,11 @@ export default {
         'moderation-list-row': {
             submission: {
                 submitted_on: `submitted on`,
-                was_accepted_on: `was accepted on`,
-                was_rejected_on: `was rejected on`,
+                accepted_on: `accepted on`,
+                rejected_on: `rejected on`,
                 submitted: `submitted`,
-                was_accepted: `was accepted`,
-                was_rejected: `was rejected`,
+                accepted: `accepted`,
+                rejected: `rejected`,
                 by: `by`,
```

Comment:
Since I'm making you do stuff with translations anyway, could you change these to be full sentences with placeholders, instead of little 1-2 word fragments? I think it'd be easier to understand, and avoids assuming english syntax in the component logic:
```
'accepted on {{date}} by {{moderatorName}}'
'accepted {{timeAgo}} by {{moderatorName}}'
'accepted automatically on {{date}}'
'accepted automatically {{timeAgo}}'
'rejected on {{date}} by {{moderatorName}}'
'rejected {{timeAgo}} by {{moderatorName}}'
'submitted on {{date}} by'
'submitted {{timeAgo}} by'
```